src/main/kotlin/io/snyk/plugin/analytics/internal/Snyk.kt

```diff
@@ -0,0 +1,48 @@
+package io.snyk.plugin.analytics.internal
+
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import java.util.concurrent.TimeUnit
+
+/**
+ * Main entrypoint for using Snyk API.
+ */
+class Snyk private constructor(
```

Comment:
Just as a suggestion: maybe  this name is too wide mining? Maybe something like something SnykSegment or SnykAnalytics will be more specific?

Reply:
Good catch, totally agree with you.

Reply:
resolved with d1b7bb2